demos/travel-and-hospitality/alliance-ticket-sharing/resources/iberia-tickets.gql

```diff
@@ -0,0 +1,288 @@
+                        precision: 5
+                    },
+                    classOfService: "J",
+                    soldAirlineDesignator: "IA",
+                    operatingAirlineDesignator: "AA",
+                    soldDestinationCityCode: "MAD",
+                    soldFlightDepartureDateTime: "2022-05-01T13:00:00Z",
+                    soldFlightNumber: "2233",
+                    soldOriginCityCode: "DCA"
+                },
+                {
+                    ticketNumber: "0014321900234",
+                    ticketingDateOfIssue: "2022-03-03",
+                    baggage: {
+                        allowance: "2PC",
+                        overAllowanceQualifier: "PC",
+                        quantity: 1,
+                        rate: {
+                            amount: 50.00,
+                            currency: "EUR",
+                            precision: 4
+                        }
+                    },
+                    couponNumber: 2,
+                    value: {
+                        amount: 325.00,
+                        currency: "EUR",
+                        precision: 5
+                    },
+                    classOfService: "J",
+                    soldAirlineDesignator: "IA",
+                    operatingAirlineDesignator: "AA",
+                    soldDestinationCityCode: "DCA",
+                    soldFlightDepartureDateTime: "2022-05-10T22:00:00Z",
+                    soldFlightNumber: "9234",
+                    soldOriginCityCode: "MAD"
+                }
+            ],
+            payments: [
+                {
+                    paymentType: CC,
+                    paymentValue: {
+                        amount: 645.20,
+                        currency: "EUR",
+                        precision: 5
+                    }
+                }
+            ],
+            prices: {
+                baseFareAmount: {
+                    amount: 575.00,
+                    currency: "EUR",
+                    precision: 5
+                }
+            }, taxes: [
+                {
+                    code: "XT",
+                    value: {
+                        amount: 50.00,
+                        currency: "EUR",
+                        precision: 4
+                    }
+                }
+            ],
+            fees: [
+                {
+                    code: XF,
+                    value: {
+                        amount: 11.20,
+                        currency: "EUR",
+                        precision: 4
+                    }
+                }, {
+                    code: MF,
+                    value: {
+                        amount: 9.00,
+                        currency: "EUR",
+                        precision: 3
+                    }
+                }
+            ]
+        },
+        aclInput: {
+            acl: [
+                { principal: {nodes: ["IberiaNode"]}, operations: [ALL, UPDATE_ACL] },
```

Comment:
I think this is the implicit behavior. Calling this out explicitly for the sake of the demo?